addons/rook/template/base/install.sh

```diff
@@ -53,6 +53,12 @@ function rook_post_init() {
     if [ "$ROOK_DID_DISABLE_EKCO_OPERATOR" = "1" ]; then
         rook_enable_ekco_operator
     fi
+
+    # wait for all pods in the rook-ceph namespace to rollout
+    log "Rook Post-init: Awaiting Rook rollout in rook-ceph namespace"
+    rook_maybe_wait_for_rollout
```

Comment:
I dont think this belongs in the post_init. it affects subsequent addons.

Reply:
I will address this in this PR: https://github.com/replicatedhq/kURL/pull/4101